src/libaktualizr/storage/embed_schemas.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import re
+
+warning_text = '''
+//This is autogenerated file, don't modify it manually
+//Latest schema timestamp: %d
+#include <string>
+#include <vector>
+'''
+
+def escape_string(sql):
+    return sql.translate(str.maketrans({'"':  r'\"', '\n':  '\\n'}))
+
+
+def apend_migration(migration_path, header_file):
+    migration_file = open(migration_path, 'r')
+    header_file.write("\"")
+    migration_content = migration_file.read()
+    header_file.write(escape_string(migration_content))
+
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
+        print("\nIncorrect arguments")
+        print("Usage:\n {} {} {} \n".format(sys.argv[0], "folder_with_schemas", "generated_file_path"))
+        sys.exit(-1)
+
+    sql_folder = sys.argv[1]
+    schemas_header = sys.argv[2]
+    migration_folder = os.path.join(sql_folder, 'migration')
+    migration_list = sorted(os.listdir(migration_folder))
+
+    max_file_stamp_path = max(migration_list, key=lambda p:os.path.getmtime(os.path.join(migration_folder,p)))
+    max_file_stamp = os.path.getmtime(os.path.join(migration_folder, max_file_stamp_path))
```

Comment:
Bonus: the script can be made to run the generation when it's itself modified. You'd need to replace these two lines with:

```
    file_depends_list = [os.path.join(migration_folder, p) for p in migration_list]
    file_depends_list.append(__file__)  # set dependency on itself (this script)
    max_file_stamp = max(os.path.getmtime(p) for p in file_depends_list)
```